.github/workflows/go.yml

```diff
@@ -2,33 +2,36 @@ name: Go
 
 on:
   push:
+    tags: [ v* ]
     branches: [ master ]
   pull_request:
     branches: [ master ]
 
-env:
-  GO111MODULE: on
+permissions:
+  # Permission for checking out code
+  contents: read
 
 jobs:
-
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ["1.13.x", "1.14.x", "1.15.x", "1.16.x", "1.17.x" ]
+        go: ["1.13.x", "1.14.x", "1.15.x", "1.16.x", "1.17.x", "1.18.x"]
```

Comment:
I think its enough to test against 1.18, 1.19, and 1.20. I think you can forget about the pre-generic versions of Go.

Reply:
Cool. I wasn't sure. (We had 1.13 in the go.mod file and I didn't want to make any big changes without some other input.)